internal/ingress/controller/config/config.go

```diff
@@ -708,7 +708,7 @@ type Configuration struct {
 
 	// DatadogSampleRate specifies sample rate for any traces created.
 	// Default: use a dynamic rate instead
-	DatadogSampleRate *float32 `json:"datadog-sample-rate",omitempty`
+	DatadogSampleRate *float32 `json:"datadog-sample-rate,omitempty"`
```

Comment:
Obviously golangci-lint works. Here is to fix the error: https://github.com/kubernetes/ingress-nginx/actions/runs/5483962701/jobs/9990894893?pr=10187

```
structtag: struct field tag `json:"datadog-sample-rate",omitempty` not compatible with reflect.StructTag.Get: key:"value" pairs not separated by spaces (govet)
```
![image](https://github.com/kubernetes/ingress-nginx/assets/30647904/ca01d5e6-c245-406c-ab8a-5e6381fbbd90)

Reply:
@dgoffredo fyi :P (my bad, didn't saw it as well but should be fixed on a followup PR)

Reply:
It's okay, that's what this action does, I will fix all lint errors in another PR.

Reply:
Nice catch! My testing didn't include serializing that `struct` as JSON.